client/web/compose/src/components/PageBlocks/Navigation/Base.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <wrap
+    v-bind="$props"
+    v-on="$listeners"
+  >
+    <div class="h-100 w-100 navigation-page-block-item">
+      <b-nav
+        :tabs="options.display.appearance === 'tabs'"
+        :pills="options.display.appearance === 'pills'"
+        :small="options.display.appearance === 'small'"
+        :fill="options.display.fillJustify === 'fill'"
+        :justify="options.display.fillJustify === 'justify'"
+        :align="options.display.alignment"
+        class="border-0 h-100"
+      >
+        <b-nav-item
```

Comment:
Why not v-for in a template and then display the correct item?

Reply:
I'm using a v-for here

Reply:
I tried v-for in a template but there is an error returned by vue, I was trying to implement something like the field editor but still coudlnt work